### PR TITLE
Adds trademark usage and antitrust policy document

### PR DESCRIPTION
**Pull request by jcooklin**
Adds policy documents for:
- [x] Trademark usage 
- [x] Antitrust

### CODE REVIEW

**Review by vsethi — COMMENTED** [review]
This is a great start and covers a lot of ground. Thank you for preparing it.

File: Antitrust-Policy.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# CNOE Antitrust Policy 
+
+## Introduction
+Cloud Native Operational Excellence (“CNOE”) is an alliance formed to provide recommended best practices for internal developer platforms utilizing cloud-native open source software (all such recommendations, the “Recommended Best Practices”).  CNOE is guided by the governing principles of being open, being welcoming and respectful, being transparent and accessible and driven by technical merit (“Guiding Principles”). It is CNOE’s policy to comply with all applicable state and federal antitrust and trade regulation laws, and applicable antitrust and competition laws of other countries (collectively, the “Antitrust Laws”).  
+
+This Policy applies to members of CNOE (“Members”) and all participants (“Participants”) as well as their employees, consultants, directors and officers.  All Members and Participants shall comply with all applicable rules, policies and procedures adopted by CNOE or the applicable working group, as updated from time to time.
```
Comment (nimakaviani) [inline Antitrust-Policy.md:6]: Not sure if I understand the distinction between participants and members. is "participants" meant to imply the broader group of users (who may not belong to a particular member company) engaged with the work done in CNOE? if so, the other occurrences of the term "users" should be replaced with the term "participants".

File: Antitrust-Policy.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+No Member or Participant shall enter into any agreement to prohibit or restrict any Member, Participant or non-member from establishing or adopting any alternative technologies to the Recommended Best Practices.
+
+While Members and Participants may elect to make contribute to the Recommended Best Practices, none of these contributions shall be exclusive to CNOE. Members and Participants shall be free to contribute to any other open source projects of their choosing, including contributions to Recommend Best Practices open source projects. Such contributions are voluntary. No Member or Participant shall require contributions to any open source project or the adoption of any particular open source software license.
```
Comment (nimakaviani) [inline Antitrust-Policy.md:26]:
```suggestion
While Members and Participants may elect to contribute to the Recommended Best Practices, none of these contributions shall be exclusive to CNOE. Members and Participants shall be free to contribute to any other open source projects of their choosing, including contributions to Recommend Best Practices open source projects. Such contributions are voluntary. No Member or Participant shall require contributions to any open source project or the adoption of any particular open source software license.

File: Antitrust-Policy.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+While Members and Participants may elect to make contribute to the Recommended Best Practices, none of these contributions shall be exclusive to CNOE. Members and Participants shall be free to contribute to any other open source projects of their choosing, including contributions to Recommend Best Practices open source projects. Such contributions are voluntary. No Member or Participant shall require contributions to any open source project or the adoption of any particular open source software license. 
+
+## Membership & Participation
+As part of its governing charter, CNOE shall establish and maintain reasonable requirements for CNOE membership that are consistent with the efficient and effective pursuit of CNOE’s purpose and goals, and all qualifying entities shall be admitted to join. Such membership shall be inclusive and non-discriminatory as to cloud providers, technology providers and customers of each of the foregoing. No qualifying entity shall be denied membership or removed from membership for any anti-competitive purpose.
```
[the remark this answers is not in the document]
Comment (nimakaviani) [reply Antitrust-Policy.md:29]: +1. I believe this is mentioned later in the document that "participation is open to all participants consistent with CNOE’s governing charter".  This paragraph also touches on the "qualifying entities". I believe for "qualifying entities", i.e. "participants consistent with the CNOE charter", admission to join shall be decided based on the voting mechanism already in place as part of CNOE's governing charter.

File: Antitrust-Policy.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Membership & Participation
+As part of its governing charter, CNOE shall establish and maintain reasonable requirements for CNOE membership that are consistent with the efficient and effective pursuit of CNOE’s purpose and goals, and all qualifying entities shall be admitted to join. Such membership shall be inclusive and non-discriminatory as to cloud providers, technology providers and customers of each of the foregoing. No qualifying entity shall be denied membership or removed from membership for any anti-competitive purpose. 
+
+Participation in CNOE by non-member Participants shall be broadly open and no participation shall be excluded except for breaches of the CNOE Code of Conduct and this Policy.
```
Comment (nimakaviani) [inline Antitrust-Policy.md:31]: same as above. based on what is discussed later in the document ""participation is open to all participants consistent with CNOE’s governing charter" and the CNOE governing charter includes the membership tiers document that we have been floating around.

File: Antitrust-Policy.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Recommended Best Practices shall not incorporate confidential or proprietary information or any material that infringes or misappropriates any third-party intellectual property rights. Recommended Best Practices shall not require a provider, developer, or end-user of Recommended Best Practice obtain a license or permission from any third party to use such technology except for any open source software license pursuant to which the Recommended Best Practice source code is licensed.
+
+CNOE and its working groups shall make available all draft Recommended Best Practices on the CNOE website or public repositories for a reasonable period of time prior to making formal recommendations.  CNOE and its Members welcome input from Participants and non-members and shall consider such feedback prior to making formal recommendations.  As set forth above, participation in CNOE is open to all participants consistent with CNOE’s governing charter. This expressly includes those who may be affected by a Recommended Best Practice recommended by CNOE who shall be given the opportunity to comment on the Recommended Best Practice.
```
Comment (cbleach) [inline Antitrust-Policy.md:58]: I have concerns about the operational burden that this places on the project.

File: Antitrust-Policy.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cbleach) [reply Antitrust-Policy.md:29]: This does not seem inline with the concept of membership tiers that ensure the project maintains it's focus on benefitting the core members and adopters.

File: Antitrust-Policy.md (hunk 5 above)
Comment (cbleach) [reply Antitrust-Policy.md:58]: "CNOE and its working groups shall make available all draft Recommended Best Practices on the CNOE website or public repositories for a reasonable period of time prior to making formal recommendations. "

How would this manifest in our operations?  We would need to publish technology choices for review before we made a decision?

" CNOE and its Members welcome input from Participants and non-members and shall consider such feedback prior to making formal recommendations."

Again, what kind of operation burden does this place on the project?  How would this manifest in practice?

"This expressly includes those who may be affected by a Recommended Best Practice recommended by CNOE who shall be given the opportunity to comment on the Recommended Best Practice."

Does this ability to comment include a requirements to publish dissent on decisions along with our guidance?

File: Antitrust-Policy.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+In the development and recommendation of Recommended Best Practices, CNOE will strive to (1) have an open source software first strategy for addressing internal developer platforms capabilities while prioritizing Cloud Native Computing Foundation ([CNCF](https://www.cncf.io/)) technologies over the alternatives; (2) be community driven in deciding technology choices and best practices; (3) offer tools and suggest best practices that work for Members but are also usable for the broader group of participants interested in adopting internal developer platforms best practices; (4) automate and customize infrastructure for developer needs, making the solutions and patterns flexible for adoption by the larger community of participants; and (5) share openly all the tools, patterns, and practices offered by Members organizations in such a way that the artifacts can be easily consumable by Participants and non-members.  The mission for the CNOE is to enable organizations to navigate tooling sprawl and technology churn by providing neutral and unbiased guidance on tooling and technology choices with refernce implementations and tools to deliver internal developer platforms.
```
Comment (vsethi) [inline Antitrust-Policy.md:9]:
```suggestion
In the development and recommendation of Recommended Best Practices, CNOE will strive to (1) have an open source software first strategy for addressing internal developer platforms capabilities while prioritizing Cloud Native Computing Foundation ([CNCF](https://www.cncf.io/)) technologies over the alternatives; (2) be community driven in deciding technology choices and best practices; (3) offer tools and suggest best practices that work for Members but are also usable for the broader group of participants interested in adopting internal developer platforms best practices; (4) automate and customize infrastructure for developer needs, making the solutions and patterns flexible for adoption by the larger community of participants; and (5) share openly all the tools, patterns, and practices offered by Members organizations in such a way that the artifacts can be easily consumable by Participants and non-members.  The mission of the CNOE is to enable organizations to navigate tooling sprawl and technology churn by providing neutral and unbiased guidance on tooling and technology choices with reference implementations and tools to deliver internal developer platforms. 
```

File: Antitrust-Policy.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+# CNOE Antitrust Policy 
+
+## Introduction
+Cloud Native Operational Excellence (“CNOE”) is an alliance formed to provide recommended best practices for internal developer platforms utilizing cloud-native open source software (all such recommendations, the “Recommended Best Practices”).  CNOE is guided by the governing principles of being open, being welcoming and respectful, being transparent and accessible and driven by technical merit (“Guiding Principles”). It is CNOE’s policy to comply with all applicable state and federal antitrust and trade regulation laws, and applicable antitrust and competition laws of other countries (collectively, the “Antitrust Laws”).
```
Comment (vsethi) [inline Antitrust-Policy.md:4]:
```suggestion
Cloud Native Operational Excellence (“CNOE”) is an alliance formed to provide recommended best practices for internal developer platforms utilizing cloud-native open source software (all such recommendations, the “Recommended Best Practices”).  CNOE is guided by the governing principles of being open, welcoming, respectful, transparent, accessible, and driven by technical merit (“Guiding Principles”). It is CNOE’s policy to comply with all applicable state and federal antitrust and trade regulation laws, and applicable antitrust and competition laws of other countries (collectively, the “Antitrust Laws”).  
```

File: Antitrust-Policy.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+## Compliance with Antitrust Laws
+Members and all Participants as well as their employees, consultants, directors and officers must comply with Antitrust Laws and this Policy in carrying out CNOE-related activities, whether directly or indirectly. By participating in CNOE-related activities, members and participants agree to abide this Policy and Antitrust Law.
```
Comment (vsethi) [inline Antitrust-Policy.md:12]:
```suggestion
Members and all Participants as well as their employees, consultants, directors and officers must comply with Antitrust Laws and this Policy in carrying out CNOE-related activities, whether directly or indirectly. By participating in CNOE-related activities, members and participants agree to abide by this Policy and Antitrust Law. 
```

File: Antitrust-Policy.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+## Compliance with Antitrust Laws
+Members and all Participants as well as their employees, consultants, directors and officers must comply with Antitrust Laws and this Policy in carrying out CNOE-related activities, whether directly or indirectly. By participating in CNOE-related activities, members and participants agree to abide this Policy and Antitrust Law. 
+
+In the event of any conflict or inconsistency between this Policy and Antitrust Laws, the Antitrust Laws shall control.
+
+## Freedom to Operate
+CNOE does not set standards. CNOE publishes Recommended Best Practices based upon technical merit the adoption of which is voluntary.
```
Comment (vsethi) [inline Antitrust-Policy.md:17]:
```suggestion
CNOE does not set standards. CNOE publishes Recommended Best Practices based on technical merit the adoption of which is voluntary. 
```

File: Antitrust-Policy.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+## Membership & Participation
+As part of its governing charter, CNOE shall establish and maintain reasonable requirements for CNOE membership that are consistent with the efficient and effective pursuit of CNOE’s purpose and goals, and all qualifying entities shall be admitted to join consistent with the CNOE charter. Such membership shall be inclusive and non-discriminatory as to cloud providers, technology providers and customers of each of the foregoing. No qualifying entity shall be denied membership or removed from membership for any anti-competitive purpose. 
+
+Participation in CNOE by non-member Participants shall be broadly open and no participation shall be excluded except for breaches of the CNOE Code of Conduct and this Policy.
```
Comment (vsethi) [inline Antitrust-Policy.md:31]:
```suggestion
Participation in CNOE by non-member Participants shall be broadly open. No participation shall be excluded except for breaches of the CNOE Code of Conduct, CNOE’s governing charter, and this Policy. 
```

File: Antitrust-Policy.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+## Development of Recommended Best Practices and Reference Implementations
+CNOE shall discuss and cultivate the Recommended Best Practices and Reference Implmentations consistent with this Policy, CNOE’s Guiding Principles, CNOE’s Mission Statement and not for any anti-competitive purpose or effect.  CNOE working groups shall develop and select Recommended Best Practices based on their quality, technological attributes and ability to address the needs of developer platform users. 
+
+Recommended Best Practices shall not incorporate confidential or proprietary information or any material that infringes or misappropriates any third-party intellectual property rights. Recommended Best Practices shall not require a provider, developer, or end-user of Recommended Best Practice obtain a license or permission from any third party to use such technology except for any open source software license pursuant to which the Recommended Best Practice source code is licensed.
```
Comment (vsethi) [inline Antitrust-Policy.md:56]:
```suggestion
Recommended Best Practices shall not incorporate confidential or proprietary information or any material that infringes or misappropriates any third-party intellectual property rights. Recommended Best Practices shall not require a provider, developer, or end-user of Recommended Best Practice to obtain a license or permission from any third party to use such technology except for any open-source software license under which the Recommended Best Practice source code is licensed.
```

File: Antitrust-Policy.md (hunk 5 above)
Comment (vsethi) [reply Antitrust-Policy.md:58]: I would also like to understand the reasoning behind making the draft available first before making it formal.

**Review by vsethi — APPROVED** [review]
No major concerns, just a few minor suggestions/comments. I would like to understand the requirement to make the draft available for sometime before making it formal though. We can discuss this in the next meetings.

File: Antitrust-Policy.md (hunk 5 above)
Comment (nimakaviani) [reply Antitrust-Policy.md:58]: Giving it another read, I believe making the recommended best practices into PRs that are available for the general public to potentially comment on, should satisfy this last paragraph. I dont think this snippet requires us to actively search for them or try to satisfy them.

Also as indicated already, participation in CNOE should remai `consistent with CNOE’s governing charter` which helps us navigating the decision making process

**Review by nimakaviani — APPROVED** [review]
LGTM

**Comment by nimakaviani** [comment]
@jcooklin can you please sign the commit so the DCO check passes? once done, we are good to go.

**Pull request merged**